paper.md

```diff
@@ -380,14 +404,12 @@ The limitation of these workflow engines is the requirement of having access to
 Although Popper can run workflows in the cloud using Kubernetes, it does not necessarily require access to a Kubernetes cluster for running the containerized steps of a workflow.
 Popper is not exclusively cloud-native since it does not assume the presence of a Kubernetes cluster for running workflows, but in addition to being able to work as cloud-native i.e. run workflows on Kubernetes, it can also behave as container-native in different computing environments like a local machine, SLURM and cloud VM instances over SSH.
 
-# Conclusion {#sec:conclusion}
-
-## Benefits
-
-## Challenges
-
-## Learning Curve
+# Conclusion {#sec:conclusionandfuturework}
 
-# Future Work {#sec:futurework}
+As future work, we have planned to add support for more container engines like NVIDIA Pyxis, Charliecloud, Shifter and resource managers like HTCondor, TORQUE to Popper in order to extend the range of the different computing environments currently supported.
+One of the future goals of Popper is to add language parsers to support running workflows written in different workflow languages like CWL, HCL, WDL, etc.
+This would allow users familiar with different workflow languages to start using Popper without making any changes to their previously written workflows.
```

Comment:
Since Popper is less expressive than CWL et al., it would be difficult to do this. The opposite would be true though (being able to run Popper workflows on engines that support CWL by transparently translating to these langauges).